src/lib/utils/application-commands/ApplicationCommandRegistry.ts

```diff
@@ -197,6 +200,12 @@ export class ApplicationCommandRegistry {
 			return;
 		}
 
+		if (getDefaultBehaviorWhenNotIdentical() === RegisterBehavior.BulkOverwrite) {
+			throw new RangeError(
+				`"runAPICalls" was called for "${this.commandName}" but the default behavior when not identical is "BulkOverwrite". This should not happen.`
+			);
```

Comment:
default === BulkOverwrite

yet the log says "but the default was not bulk overwrite"

These statements conflict

Reply:
You didn't read it right...